internal/version/version.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+const (
+	// Version is the version of abc-updater.
+	Version = "0.0.1"
```

Comment:
Why not grab using the github.com/abcxyz/pkg/buildinfo as done in https://github.com/abcxyz/abc/blob/77cd3b54446d852a7b569ccec872fb7d5e3917e8/internal/version/version.go#L3

Reply:
Because, at least for this part, we don't control the "build" of the client. The buildinfo pattern requires that we control the build to inject the information.

Reply:
That makes sense, it will be looking at the version for the consuming application rather than the library.

We could make a CI/CD pipeline to update the value, but never changing version seems to be the industry-accepted way of handling user agents (See Mozilla/5.0)

Reply:
> We could make a CI/CD pipeline to update the value, but never changing version seems to be the industry-accepted way of handling user agents (See Mozilla/5.0)

Lol yes... changing the version might break someone!